app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/BackEndCommHandler.java

```diff
@@ -0,0 +1,15 @@
+package br.com.nespressohelper.daitan.nespressohelper;
+
+/**
+ * Created by estrela on 12/27/17.
+ */
+
+public interface BackEndCommHandler {
+
+    enum Method{GET, POST}
+
+    void handleResponse(String response, Method m);
```

Comment:
You could maybe create a single method that receives code and the message body (if any) instead of having two methods. Once we start covering network error cases the response code might be relevant even if we are doing a GET request.